src/decodeUint8FromString.js

```diff
@@ -4,19 +4,22 @@
  */
 
 // Adapted from https://stackoverflow.com/a/36046727
-const decodeUint8FromString = str => {
+const decodeUint8FromString = (str, format = 'base64') => {
   if (typeof str !== 'string') {
     throw new TypeError('Input must be a string!')
   }
   if (str.length < 50000) {
+    if (format !== 'base64' && format !== 'hex') {
+      throw new TypeError('Input must be formatted as a base64 or hex string!')
+    }
```

Comment:
```suggestion
  }
  if (format !== 'base64' && format !== 'hex') {
    throw new TypeError('Input must be formatted as a base64 or hex string!')
  }
  if (str.length < 50000) {
```
Moved this outside the length check. We can only run regex on things less than 50000 but the format check can run always.